client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/useInteractionsForm.ts

```diff
@@ -68,7 +75,11 @@ const useInteractionsForm = (props : useInteractionFormIncome): useInteractionFo
         axios.post('/intersections/createContactEvent', {
           ...interactionsDataToSave,
           [InteractionEventDialogFields.LOCATION_ADDRESS]: locationAddress,
-          [InteractionEventDialogFields.INVESTIGATION_ID]: epidemiologyNumber
+          [InteractionEventDialogFields.INVESTIGATION_ID]: epidemiologyNumber,
+          contacts: interactionsDataToSave.contacts.map((contact: Contact) => ({
```

Comment:
why don't you use interactionEventDialogFields to get 'contacts'?

Reply:
does it make a diffrence?

Reply:
No, but you should use it.
think that if the field name changes , you will need to make changes only once